pkg/scheduler/api/compatibility/compatibility_test.go

```diff
@@ -1218,34 +1224,48 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 		if !reflect.DeepEqual(policy, tc.ExpectedPolicy) {
 			t.Errorf("%s: Expected:\n\t%#v\nGot:\n\t%#v", v, tc.ExpectedPolicy, policy)
 		}
-
+		policyConfigMap := v1.ConfigMap{
```

Comment:
I don't think we need to convert `tc.JSON` lines above to a policy anymore. Verifying `ExpectedPolicy` seems out of scope for this unit test.
Verifying the parsing is a good test for `Codec` in `pkg/scheduler/api/latest`, not this.
Instead, here we should verifying that `scheduler.Algorithm` corresponds to the policy.

Reply:
@ahg-g, any thoughts?

Reply:
+1 if we can do that, that is conformance.